.buildkite/scripts/pipelines/security_solution_quality_gate/pipeline.sh

```diff
@@ -13,7 +13,6 @@ echo "--- Serverless Security Second Quality Gate"
 cd x-pack/test/security_solution_cypress
 set +e
 
-VAULT_DEC_KEY=$(vault read -field=key secret/kibana-issues/dev/security-solution-qg-enc-key)
-ENV_PWD=$(echo $TEST_ENV_PWD | openssl aes-256-cbc -d -a -pass pass:$VAULT_DEC_KEY)
+QA_API_KEY=$(retry 5 5 vault read -field=qa_api_key secret/kibana-issues/dev/security-solution-qg-enc-key)
 
-CYPRESS_ELASTICSEARCH_URL=$TEST_ENV_ES_URL CYPRESS_BASE_URL=$TEST_ENV_KB_URL CYPRESS_ELASTICSEARCH_USERNAME=$TEST_ENV_USERNAME CYPRESS_ELASTICSEARCH_PASSWORD=$ENV_PWD CYPRESS_KIBANA_URL=$CYPRESS_BASE_URL yarn cypress:run:qa:serverless; status=$?; yarn junit:merge || :; exit $status
\ No newline at end of file
+PARALLEL_COUNT=4 CLOUD_QA_API_KEY=$QA_API_KEY yarn cypress:run:qa:serverless:parallel; status=$?; yarn junit:merge || :; exit $status
```

Comment:
Can you please explain why was 4 selected as `parallel_count`?

Reply:
At the time being the specific PR runs the tests in a default Buildkite agent. It is extremely difficult to guess how many threads can be running in this agent. So a minimum acceptable number is hard coded here. 

In a next PR for the [relevant ticket](https://github.com/elastic/security-team/issues/7962) this will be addressed and a totally similar way like the [base.yml](https://github.com/elastic/kibana/blob/main/.buildkite/pipelines/pull_request/base.yml) will be followed.